File: eRestorante/eRestorante.Models/Model/Drink.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eRestorante.Models.Model
{
    public partial class Drink
    {
        public int DrinkId { get; set; }

        public string DrinkName { get; set; } = null!;

        public string DrinkDescription { get; set; } = null!;

        public decimal DrinkCost { get; set; }

        public decimal? DrinkAlcoholPerc { get; set; }

        public byte[]? DrinkImage { get; set; }

        public int? CategoryId { get; set; }
    }
}

File: eRestorante/eRestoranteUI/erestorante_desktop/lib/models/drink.dart
import 'dart:ffi';

import 'package:json_annotation/json_annotation.dart';

part 'drink.g.dart';

@JsonSerializable()
class Drink{
  int? drinkId;
  String? drinkName;
  String? drinkDescription;
  double? drinkCost;
  int? drinkAlcoholPerc;
  String? drinkImage;
  int? categoryId;

  Drink(this.drinkId, this.drinkName, this.drinkDescription, this.drinkCost, this.drinkAlcoholPerc, this.drinkImage, this.categoryId);
  factory Drink.fromJson(Map<String, dynamic> json) => _$DrinkFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$DrinkToJson(this);
}